com.playeveryware.eos/Runtime/Core/DLLHandle.cs

```diff
@@ -20,8 +20,6 @@
 * SOFTWARE.
 */
 
-﻿//#define ENABLE_DLLHANDLE_PRINT
```

Comment:
This should go back, but perhaps with a comment explaining what it does.

Reply:
Done! :)